.travis.yml

```
@@ -111,6 +111,12 @@ jobs:
       - bin/s3 cp myst_ubuntu.tgz s3://travis-$TRAVIS_BUILD_NUMBER/docker-images
       name: "Ubuntu docker image"
 
+    - script:
+      - go get github.com/go-swagger/go-swagger/cmd/swagger/
+      - bin/swagger_generate
+      - bin/s3 cp tequilapi.json s3://travis-$TRAVIS_BUILD_NUMBER/build-artifacts
```

```suggestion
      - bin/s3 cp tequilapi.json s3://travis-$TRAVIS_BUILD_NUMBER/build-artifacts/
```
and `/` at the end, or it will create file `/build-artifacts` instead of directory.